install_config/aggregate_logging.adoc

```diff
@@ -676,6 +675,11 @@ This loopback must be maintained manually outside of {product-title}, on the
 node. You must not maintain it from inside a container.
 ====
 
+You can use Container-Native Storage (CNS) 3.6 to enable Gluster Block Storage
+as a backend for logging with ElasticSearch. For more information, see
+link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/apes03[Enabling
```

Comment:
This link should point to the example I'm writing on how to structure the Advanced Installer inventory file to enable this. Now sure how that should be handled, however.